lib/mongo_oplog_backup/config.rb

```diff
@@ -65,11 +74,11 @@ def exec(cmd)
     end
 
     def mongodump(*args)
-      exec(['mongodump'] + command_line_options + args.flatten)
+      exec(['/usr/local/Cellar/mongodb/3.2.11/bin/mongodump'] + command_line_options + args.flatten)
```

Comment:
I'm guessing this should either be configurable or just plain `mongodump`? Same with the `mongo` command below.